galaxy_ng/app/api/ui/viewsets/group.py

```diff
@@ -29,6 +29,11 @@ class GroupViewSet(LocalSettingsMixin, viewsets.GroupViewSet):
     filterset_class = GroupFilter
     permission_classes = [access_policy.GroupAccessPolicy]
 
+    # override attribute of parent class
+    def _remove_attr(self):
+        raise AttributeError
+    queryset_filtering_required_permission = property(_remove_attr)
```

Comment:
This will make all groups visible to everyone (able to call the list endpoint.)

Reply:
We use permissions like `galaxy.delete_group` not `auth.delete_group` (which is why we needed to override this, it seems to now require use of `auth.view_group` - but correct me if i am mistaken), and in this case `GroupViewSet` `list` is already visible to everyone: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/access_control/statements/standalone.py#L176-L180